pkg/integrations/v4/emitter/emitter.go

```diff
@@ -64,7 +66,7 @@ func (e *VersionAwareEmitter) Emit(definition integration.Definition, extraLabel
 		fields["parent_integration_name"] = definition.CfgProtocol.ParentName
 	}
 
-	elog.WithFields(fields).Debug("Received payload.")
+	elog.WithField("env", helpers.ObfuscateSensitiveDataFromMap(definition.ExecutorConfig.Environment)).WithFields(fields).Debug("Received payload.")
```

Comment:
isn't this line a bit complex? could we split it adding a variable for `helpers.ObfuscateSensitiveDataFromMap(definition.ExecutorConfig.Environment)`

Reply:
ok, I can do it like that will increase the visibility sure! and then I'll also squash the commits